examples/all-clusters-app/esp32/sdkconfig.defaults

```diff
@@ -44,3 +44,7 @@ CONFIG_DEVICE_PRODUCT_ID=0x4541
 
 #enable debug shell
 CONFIG_ENABLE_CHIP_SHELL=y
+
+#enable lwIP route hooks
+CONFIG_LWIP_HOOK_IP6_ROUTE_DEFAULT=y
+CONFIG_LWIP_HOOK_ND6_GET_GW_DEFAULT=y
```

Comment:
Should this also be added in `sdkconfig_c3devkit.defaults`, `sdkconfig_m5stack.defaults` and any relevant sdkconfigs?